tests/examples/redirected.css

```diff
@@ -0,0 +1 @@
+p { color: violet }
```

Comment:
I just renamed `redirect.css` to `redirected.css` because the name `redirect.css` sounds like it's *going to* redirect, when in fact it's more like the "destination".